library/data/loot_table/src/main/java/org/quiltmc/qsl/loot/api/QuiltLootTableBuilder.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019 FabricMC
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.loot.api;
+
+import net.minecraft.loot.LootPool;
+import net.minecraft.loot.LootTable;
+import net.minecraft.loot.context.LootContextType;
+import net.minecraft.loot.function.LootFunction;
+import org.quiltmc.qsl.loot.mixin.LootTableBuilderHooks;
+
+import java.util.Collection;
+
+public class QuiltLootTableBuilder extends LootTable.Builder {
```

Comment:
Javadoc would be nice

Reply:
Documented QuiltLootPoolBuilder and this